# * Set node to accept newer versions to fulfill dep
# * Select node-gyp versions with python3 support
#     - https://github.com/sass/node-sass/pull/2841
#     - https://github.com/sass/node-sass/issues/2716
--- package.json.orig	2021-04-15 21:50:37.757593564 -0400
+++ package.json	2021-04-15 21:51:37.819672207 -0400
@@ -251,7 +251,7 @@
     "jsdoc": "3.6.2",
     "mocha": "4.1.0",
     "mocha-testcheck": "1.0.0-rc.0",
-    "node-gyp": "5.0.3",
+    "node-gyp": "6.1.0",
     "node-sass": "4.14.1",
     "node-sass-import-once": "1.2.0",
     "npm-run-all": "4.1.5",
@@ -273,11 +273,11 @@
     "webpack-cli": "3.3.7",
     "webpack-dev-server": "3.8.0"
   },
-  "resolutions": {
+  "resolutions": {"node-sass/node-gyp": "^6.0.0",
     "@storybook/react/@storybook/core/node-fetch": "2.6.1"
   },
   "engines": {
-    "node": "12.18.3"
+    "node": ">=12.18.3"
   },
   "build": {
     "appId": "org.whispersystems.signal-desktop",
